Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:22-alpine
+
+# App working directory
+#
+WORKDIR /app
+
+# Copy over the package files to the app working directory
+#
+COPY package.json package-lock.json ./
+
+# Chromium is needed for some testing libraries, but is not available on
+# ARM64 architecture (i.e., Apple Silicon). Set a flag to skip the chromium
+# download unless the current architecture is ARM64.
+#
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
```

Comment:
you laid this out in the PR notes. do you want to default this to `true` or `false`? which is least risky? are you sure that skipping the download will result in successful cases even if the user doesn't match your test on line 17?

Reply:
I defaulted this to true based on the examples I saw when researching this issue. With that said, it makes sense to me. When building the image on our laptops, we know that our macs use arm64. But, the architecture for running the github actions (which will also build the image) will likely not be arm64. Regardless, the if/then logic check has to happen to flip the skip switch whether or not the ENV var is initialized to true or false.

Reference: https://github.com/puppeteer/puppeteer/issues/7740